students/yeseul/users/views.py

```diff
@@ -41,3 +41,22 @@ def post(self, request):
         
         return JsonResponse({'message' : 'SUCCESS'}, status = 201)
 
+class LoginView(View):
+    def post(self, request):
+        try:
+            request_data = json.loads(request.body)
+            email        = request_data['email']
+            password     = request_data['password']
+
+            if not User.objects.get(email=email).password == password:
+                return JsonResponse({'message' : 'INVALID_USER'}, status = 401)
```

Comment:
존재 유무만 판단할 때에는 `.exists()`를 사용하는 것이 성능, 가독성 면에서 좋습니다.
```suggestion
            if not User.objects.filter(email=email, password = password).exists():
                return JsonResponse({'message' : 'INVALID_USER'}, status = 401)
```

Reply:
- exists()로 수정했습니다.
- ObjcetDoesNotExist 에러를 캐치하는 부분은 삭제했습니다.